Documentation/ceph-csi-drivers.md

```diff
@@ -0,0 +1,382 @@
+# Running ceph CSI drivers with Rook
+
+Here is a guide on how to implement ceph-csi drivers on a Kubernetes cluster with Rook. The way to implement it should change relatively soon as there is an effort to implement a csi plugin within Rook directly ([#1385](https://github.com/rook/rook/issues/1385), [#2059](https://github.com/rook/rook/pull/2059)).
+
```

Comment:
since there are different major topics, we might want links at the top for enabling CSI, consuming CSI, and taking snapshots. For example, see the links at the top of [this page](https://rook.io/docs/rook/v0.8/advanced-configuration.html)